### ISSUE TITLE
Please provide aarch64 releases too

### ISSUE THREAD

**Issue by andre2007** [Pure-D/serve-d#90]
I am using VSC on my raspberry 3 (aarch64 switched on).
Compiling serve-d is not possible. Limited memory will cause system freeze even with unlimited swap switched on.
If possible could you provide also aarch64 release artifacts?

**Comment by WebFreak001** [comment]
hm it might be possible to compile natively using self-hosted runners, however I also don't have anything more powerful than a raspberry pi 3 for aarch64 compilation (and I have bad experience with LDC cross compilation never working)

**Comment by Cogitri** [comment]
I happen to have an Aarch64 container that's more than capable of providing those, but I'd only be able to compile it with GDC since LDC isn't available on aarch64 on Alpine Linux

**Comment by WebFreak001** [comment]
I will first try to look into cross compiling once more using GitHub actions, but wouldn't the linux-aarch64 asset of LDC work on alpine linux? @Cogitri

**Comment by Cogitri** [comment]
No, that's compiled against glibc and not musl. Apparently ldc is a little unstable on Aarch64 anyway, so maybe that's why it crashed for us on Alpine.
